recipe/magento2.php

```diff
@@ -443,7 +440,11 @@ function magentoDeployAssetsSplit(string $area)
 /**
  * Update cache id_prefix on deploy so that you are compiling against a fresh cache
  * Reference Issue: https://github.com/davidalger/capistrano-magento2/issues/151
- * use set('use_redis_cache_id') in your deployer script to enable
+ * use add the following to your deployer scripts:
```

Comment:
Grammar: "To use this feature, add the following to your deployer scripts:"

Reply:
Thanks @schmengler 
That's now updated

Reply:
"use add the following [...]" is still an incomplete sentence. If you want to keep it short: "Usage: add the following [...]"